helpers/argumentParsers.go

```diff
@@ -1705,3 +1707,7 @@ func ParseGetSocketOption(rawValue uint64) (SocketOptionArgument, error) {
 	}
 	return v, nil
 }
+
+func ParseBpfProgType(rawValue uint64) (libbpfgo.BPFProgType, error) {
+	return libbpfgo.BPFProgType(rawValue), nil
```

Comment:
We should have a check here to make sure that this is a valid BPF program type. For example, `ParseSocketDomainArgument` has a map of known types. We could implement something similar in libbpfgo.go. What do you think?

Reply:
i agree. it's more correct to have this check here.
in the `String()` function of `libbpfgo.BPFProgType` we have something like that, but the helper parsing function should return an error if the value is not a valid BPF program type.

i'll push an update soon